environment.yml

```
@@ -27,10 +27,8 @@ dependencies:
   - pip:
     - opencv-contrib-python
     - opencv-python-headless
-    - --find-links https://download.pytorch.org/whl/torch_stable.html
-    - torch==1.8.1+cpu; sys_platform != "darwin"
-    - torchvision==0.9.1+cpu; sys_platform != "darwin"
-    - torch==1.8.1;sys_platform == "darwin"
-    - torchvision==0.9.1; sys_platform == "darwin"
+    # PyTorch's Linux distribution is very large due to its GPU support,
+    # but we only need that for training models. Our users only need CPU
+    - --extra-index-url https://download.pytorch.org/whl/cpu
     - protobuf==3.20.1
-    - ivadomed==2.9.5
+    - ivadomed@git+https://github.com/ivadomed/ivadomed@ng/cu113
```

```suggestion
    - ivadomed==2.9.6
```
(to remember to change this after the new release)